clients/include/testing_syrkx_strided_batched.hpp

```diff
@@ -53,101 +48,171 @@ hipblasStatus_t testing_syrkx_strided_batched(const Arguments& argus)
         return HIPBLAS_STATUS_SUCCESS;
     }
 
+    int           K1       = transA == HIPBLAS_OP_N ? K : N;
+    hipblasStride stride_A = lda * K1 * stride_scale;
+    hipblasStride stride_B = ldb * K1 * stride_scale;
+    hipblasStride stride_C = ldc * N * stride_scale;
```

Comment:
I think this is pre-existing overflow potential, either size_t(lda) or move the stride_scale double to the front to do all math as double.